website/blog/2023-07-27-grafana-monitoring-ferretdb.md

```diff
@@ -0,0 +1,274 @@
+---
+slug: grafana-monitoring-ferretdb
+title: 'Grafana Monitoring and Visualization for FerretDB'
+authors: [alex]
+description: >
+  FerretDB is well suited to leverage the MongoDB data source plugin for Grafana to monitor, analyze, and visualize your data.
+image: /img/blog/ferretdb-grafana.jpg
+tags: [compatible applications, tutorial]
+---
+
+![Grafana Monitoring for FerretDB](/img/blog/ferretdb-grafana.jpg)
+
+Monitoring and visualization tools play an essential role in overseeing the operations of a database, and [Grafana](https://grafana.com/) is one of the most popular tools.
+
+<!--truncate-->
+
+As the truly open-source alternative to MongoDB, [FerretDB](https://www.ferretdb.io/) helps you remain in the open source ecosystem and [avoid vendor lock-in](https://blog.ferretdb.io/5-ways-to-avoid-database-vendor-lock-in/).
+It also allows you to use the same commands and queries you're used to, along with many applications that MongoDB users already know.
+
+In this regard, FerretDB is well suited to leverage the [MongoDB data source plugin for Grafana](https://grafana.com/grafana/plugins/grafana-mongodb-datasource/).
+
+This blog post will cover Grafana and how you can use it to analyze and monitor your FerretDB data.
+
+## What is Grafana?
+
+Grafana is a comprehensive visualization and monitoring tool for managing and tracking data from multiple data sources such as MongoDB, ElasticSearch, PostgreSQL, MySQL, and Graphite, among others.
+
+With insight into large data sets with complex infrastructures, Grafana provides you with configurable dashboards that you can use to tailor the different visualizations (or panels) for your data.
+Dashboards in Grafana consists of panels containing several views such as graph, table, and heatmap.
+
+Besides this, Grafana offers many data source plugins – including the MongoDB plugin for Grafana – that can plug into your existing data.
+This gives you direct access to your data in real time and enables you to visualize and build dashboards for your contextual metrics.
+
+And since there's a MongoDB plugin for Grafana, you can also set up Grafana to work with FerretDB databases, without any hassles.
+
+## What is FerretDB and How is it Different from MongoDB?
+
+FerretDB is a open source document database alternative to MongoDB that converts MongoDB wire protocol queries to SQL, with PostgreSQL and SQLite (Beta version) as the backend.
+
+By using FerretDB instead of MongoDB, you get the following benefits:
+
+- **Open source:** FerretDB is completely open source, released under the Apache 2.0 license – anyone can use it, modify it, and contribute to it.
+  On the other hand, MongoDB is no longer open source after changing their license to Server side Public License (SSPL), which leaves room for different interpretation.
+  For example, you may need to make your source code available if you provide MongoDB as a service.
```

Comment:
"For example, you may need to make your source code available if you provide MongoDB as a service. Also, the license does not allow you to embed MongoDB into your application."